src/modules/monsterpi/filesystem/home/pi/fdm-monster-daemon/update-fdm-monster.sh

```diff
@@ -1,2 +1,61 @@
 #!/bin/bash
+: '
+/**
+ * Created by D. Zwart
+ * Description: Performs all the steps to update FDM Monster
+ * Last change: Update is running from the known server/daemon folders with push/popd
+ * v1.1
+ * 05/05/2023
+ */
+'
 
+# If you are logging in as pi user, you can skip the sudo su pi command.
+# I decided to switch to the pi user with sudo su pi beforehand as I was logging in as david.
+# Then I went to the folder /home/pi/fdm-monster-daemon/ with the command cd ~/fdm-monster-daemon.
+# Then I ran the script I made sudo ./update-fdm-monster.sh.
+
+# Step 0) Ensure we are root.
+set -e
+if [ "$EUID" -ne 0 ]; then
+  echo "Please run as root, this script must reinstall a service and, therefore, requires elevated permissions (root)"
+  exit
+fi
+
+# Step 0b) Set the variables needed later
+ts=6 # total steps
+org=fdm-monster
+repo=fdm-monster
+server_path="/home/pi/fdm-monster/server/"
```

Comment:
Use relative path here. 
```suggestion
server_path="~/fdm-monster/server/"
```